tcl/Exception.i

```diff
@@ -20,6 +20,13 @@
     fprintf(stderr, "Error: out of memory.\n");
     exit(1);
   }
+  catch (ExceptionMsg &excp) {
+    if (!excp.suppressed()) {
+      Tcl_ResetResult(interp);
+      Tcl_AppendResult(interp, "Error: ", excp.what(), nullptr);
+      return TCL_ERROR;
```

Comment:
The whole point of errors are to throw out of the command because it literally means the 
command cannot continue.
You can suppress the mssage, but not the throw.

Reply:
Sorry, fixed